root/vars.js

```diff
@@ -21,7 +21,9 @@ declare var GLOBAL_DBDEFS_NAMESPACE: '__MB_DBDefs__';
 declare var addColon: (variable: Expand2ReactInput) => Expand2ReactOutput;
 declare var addColonText: (variable: StrOrNum) => string;
 declare var hyphenateTitle: (title: string, subtitle: string) => string;
-
+function nonEmpty(value: mixed): boolean %checks {
+  return value !== null && value !== undefined && value !== '';
+}
```

Comment:
Does this work without an accompanying `declare var`? I thought it wouldn't make it global without that. I guess you can tell by if vscode shows the correct function type on hover rather than just `any`.